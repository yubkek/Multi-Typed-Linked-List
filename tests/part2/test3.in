NEW 3
1
c
32e2
NEW 3
1
2
3
DELETE 0 0
DELETE -1 1
DELETE 0 -1
DELETE 0 -1