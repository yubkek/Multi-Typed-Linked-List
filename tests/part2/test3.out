List 0: 1 -> c -> 3200.00
List 1: 1 -> 2 -> 3
List 0: c -> 3200.00
INVALID COMMAND: DELETE
List 0: c
List 0: 